src/main/java/org/calypsonet/terminal/calypso/transaction/CardTransactionManager.java

```diff
@@ -329,7 +347,7 @@ CardTransactionManager prepareReadRecords(
    * @throws IllegalArgumentException If one of the provided argument is out of range.
    * @since 1.1.0
    */
-  CardTransactionManager prepareReadRecordMultiple(
+  CardTransactionManager prepareReadPartialRecords(
```

Comment:
to precise the int value of the offset to set it at the first byte of the record: 0 or 1?